rachelekm/vegishphilly
# Adding Simple JWT to Django

## Overview

This PR sets up the basic features of a JSON Web Token authentication backend through Simple JWT. An access and refresh token will be returned in response to POST request to `/api/token/` with proper login credentials.

### Notes

Simple web token auth fits the needs for now, but does not include password reset functionality

## Testing Instructions

 * Check out this branch and `update`
 * If you have no user, `scripts/manage createsuperuser`
 * Use curl to test with your new user credentials:
```
 curl \
  -X POST \
  -H "Content-Type: application/json" \
  -d '{"username": "YOUR_USERNAME", "password": "YOUR_PASSWORD"}' \
  http://localhost:8033/api/token/
```
* Response should include refresh and access tokens
